File: src/test/java/com/tom/PersonTest.java
package com.tom;

import com.tom.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class PersonTest {
    @Test
    public void bmiTest(){
        Person person = new Person(66.5f,1.7f);
        Assertions.assertEquals(66.5f/(1.7f*1.7f),person.bmi());
    }
    @Test
    public void getAverageTest(){
        Student student = new Student("1","Tom",60,49);
        Assertions.assertEquals((60+49)/2,student.getAverage());
    }
}

File: src/test/kotlin/com/tom/PersonTest.kt
package com.tom.kotlin

import com.tom.Human
import com.tom.Student
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class PersonTest {
    @Test
    fun bmiTest(){
        val human = Human(60.5f,1.7f,"Peter")
        Assertions.assertEquals(60.5f/(1.7f*1.7f),human.bmi())
    }
    @Test
    fun getAverageTest(){
        val student = Student("Peter",60,70)
        Assertions.assertEquals((60+70)/2,student.getAverage())
    }
    @Test
    fun highestTest(){
        val student = Student("Peter",60,70)
        Assertions.assertEquals(70,student.highest())
    }



}